src/Parser/HavocDemonHunter/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+
+import { formatPercentage } from 'common/format';
+
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static ABILITIES_ON_GCD = [
```

Comment:
Are those all abilities? If you're missing any your ABC times will be too high.

Reply:
This is all the abilities that trigger the GCD :)